### ISSUE TITLE
Update installation instructions in README.md

### ISSUE THREAD

**Issue by notmessenger** [softlayer/sl-ember-components#234]
While running the `ember install:addon sl-ember-components` does install this add on there are additional steps you must take in your application to fully utilize it.  These include:

* Need to run `npm install --save-dev ember-cli-less`
* Need to modify the Brocfile.js file to add:
```
app.import({
    development : 'bower_components/bootstrap/dist/js/bootstrap.js',
    production  : 'bower_components/bootstrap/dist/js/bootstrap.min.js'
});
```
* Create the app/styles/app.less file and add to it:
```
@import 'sl-ember-components';
```

Add these additional steps to the README.md file.

This also relates to https://github.com/softlayer/sl-ember-components/issues/47

**Comment by notmessenger** [comment]
For the 2nd bullet point, can we possibly use app.import in "included" in index.js to remove needing to do this in the consuming application?

**Comment by notmessenger** [comment]
Created https://github.com/softlayer/sl-ember-components/issues/281 to capture the comment above about using "included"

**Comment by jdhines** [comment]
I know this is closed, but not sure of the appropriate place to ask for help. I was already using SASS stylesheets in my app (and the broccoli-sass package for compiling app.scss). When I follow the instructions above, then I have app.less in addition to app.scss. But when I run `ember server` I get a broccoli error saying it can't find the app.less file.

**Comment by notmessenger** [comment]
@jdhines Unfortunately we have done any work with SASS with Ember CLI yet, and certainly not a combination of both of them in the same application.  At the moment I do not have any helpful information I can pass along.  Completely un-researched thoughts here but I wonder if the new Addon Nesting support of Ember CLI 0.2.0 would lead to a resolution of this issue.  We are planning on updating this Addon to 0.2.0 in the very near future (assuming any incompatibilities with Ember 1.10 does not delay the work)

**Comment by deafff** [comment]
Is there any ETA on update to 0.2.0 and addon nesting? This problem with sass+less is breaking my app as well.

**Comment by notmessenger** [comment]
@deafff @jdhines - @joshforisha Currently has a non-published branch he's working on that upgrades this repo to Ember CLI 0.2.1 that we are putting through it's final paces.  I anticipate being able to release it next week sometime.

**Comment by notmessenger** [comment]
@deafff @jdhines - @joshforisha has been working on the upgrade to Ember CLI 0.2.3 and has completed most of it but during the upgrade we have determined that we need to re-tool the implementation of the grid to be compatible with HTML Bars, which is what is currently holding up the upgrade and release.  Just wanted to give you an update.

**Comment by notmessenger** [comment]
@deafff @jdhines https://github.com/softlayer/sl-ember-components/pull/344 has been merged in, which upgrades this repository to Ember CLI 0.2.7.  We will be tagging a new release of this repository very soon.

**Comment by notmessenger** [comment]
@deafff @jdhines The https://github.com/softlayer/sl-ember-components/tree/v0.11.0 branch of this repo contains changes that isolates the repo's use of LESS from the consuming application so that all that the repo provides to the consuming application is compiled CSS, allowing the exact configuration you were asking for.  See https://github.com/softlayer/sl-ember-components/blob/v0.11.0/README.md for more information and https://github.com/softlayer/sl-ember-components/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22v0.11.0+%28Bootstrap+Styling%29%22 to track when all the issues in this milestone are resolved and v0.11.0 is released.
